### PR TITLE
Misc updates to setup.py, check.py and weekly ci

### PR DESCRIPTION
**Pull request by ana**
I have grouped 3 different changes in the same PR because I think they're relatively trivial. Tell me if you find it's a bad idea :-)

At bit of context to the changes in `check.py`. While I don't expect to use the disable options often in our internal CI, they are useful locally when debugging things.

### CODE REVIEW

File: selftests/check.py [hunk 1]
```diff
@@ -201,6 +201,22 @@ def parse_args():
     parser.add_argument('--disable-plugin-checks',
                         help='Disable checks for a plugin (by directory name)',
                         action='append', default=[])
+    parser.add_argument('--disable-selftests-nrunner',
```
Comment (willianrampazzo) [inline selftests/check.py:204]: We need a better naming here. I had the impression it was disabling all the `nrunner` tests, but that is not true.

File: .github/workflows/weekly.yml [hunk 2]
```diff
@@ -46,5 +46,5 @@ jobs:
         env:
           AVOCADO_LOG_DEBUG: "yes"
           AVOCADO_CHECK_LEVEL: "1"
-        run: make check
+        run: python selftests/check.py --disable-plugin-checks robot
```
Comment (willianrampazzo) [inline .github/workflows/weekly.yml:49]: I see `develop` is executed above. All is good here.

File: selftests/check.py (hunk 1 above)
Comment (ana) [reply selftests/check.py:204]: Good point. I'm going to rename it to `--disable-selftests-nrunner-interface`,

**Review by willianrampazzo — APPROVED** [review]

**Review by richtja — APPROVED** [review]
LGTM, thanks

**Review by clebergnu — APPROVED** [review]
LGTM, thanks!

**Pull request merged**